EIPS/eip-1.md

```diff
@@ -451,7 +462,7 @@ If the EIP isn't ready, the editor will send it back to the author for revision,
 
 Once the EIP is ready for the repository, the EIP editor will:
 
-- Assign an EIP number (generally the PR number, but the decision is with the editors)
+- Assign an EIP number
```

Comment:
```suggestion
- Assign an EIP number (generally incremental, but the decision is with the editors, number sniping is not allowed)
```
I'm for keeping it simple.  
* Numbers are generally incremental and going to be sequential moving forward. (Manually assigned or automatically assigned)
* EIP editors can manually assign differently where required (generally number sniping).
* Number sniping is not allowed.

This is hopefully an acceptable compromise

Reply:
This seems good to me.

Reply:
```suggestion
- Assign an EIP number (generally incremental; editors can reassign if number sniping is suspected)
```
I do have a quick question - how on earth is number sniping supposed to work if editors are the ones choosing when to merge PRs?

Reply:
Sniping example: a meme-r creates a PR as soon as a meme number is available (perhaps using a bot or a spam EIP/ERC) but the PR EIP/ERC doesn't have a spec. 

Meme-r says they should get the meme number. 

Editors say no as EIP-1 says no sniping. 

EIP/ERC is assigned a manual number (incremented if needed to not reward sniping) once the PR has a spec.